.circleci/config.yml

```diff
@@ -68,22 +68,32 @@ jobs:
         name: create ilastik conda environment
         command: >
             cd ${ILASTIK_ROOT}/ilastik-meta &&
-            mamba env create --name ${TEST_ENV_NAME} --file ilastik/dev/environment-dev.yml &&
-            mamba install --name ${TEST_ENV_NAME} --freeze-installed -c ilastik-forge -c conda-forge mpi4py pytest-cov volumina &&
-            mamba run -n ${TEST_ENV_NAME} pip install -e ilastik
+            conda env create --name ${TEST_ENV_NAME} --file ilastik/dev/environment-dev.yml &&
+            conda install --name ${TEST_ENV_NAME} --freeze-installed -c ilastik-forge -c conda-forge mpi4py pytest-cov volumina &&
+            conda run -n ${TEST_ENV_NAME} pip install -e ilastik
 
+    - run:
+        name: start xvfb
+        command: >
+          Xvfb :42 -screen 0 1024x768x24
```

Comment:
Is 42 a magic number here (and below for `DISPLAY`) or arbitrary?

Reply:
it's random